frontend/src/components/AdviceBar.tsx

```diff
@@ -0,0 +1,34 @@
+//도움말 띄울 스낵바
+import { Button, Snackbar } from '@mui/material';
+import styled from 'styled-components';
+
+interface snackBarProps {
+  advice: string; //도움말 내용
+  isOpen: boolean;
+  onClose: () => void; // 닫기 버튼 클릭 시 호출될 함수
+}
+
+const AdviceBar: React.FC<snackBarProps> = ({ advice, isOpen, onClose }) => {
```

Comment:
최근에는 React.FC 보다 아래와 같이 함수형으로 타입스크립트 많이 적용하는데 그 구조로 수정도 가능할까요~?
[공식문서](https://ko.react.dev/learn/typescript)도 참고차 살펴보셔요~ㅎㅎ
```suggestion
function AdviceBar({ advice, isOpen, onClose }: SnackBarProps) {
```


+) 추가로 컴포넌트명은 `AdviceBar`인데 interface는 `SnackBarProps`네요? 네이밍 통일해주세요~